docs/book/src/reference/jobs.md

```diff
@@ -19,12 +19,18 @@ Prow Presubmits:
 * [pull-cluster-api-verify-main] `./scripts/ci-verify.sh`
 * [pull-cluster-api-test-main] `./scripts/ci-test.sh`
 * [pull-cluster-api-test-mink8s-main] `./scripts/ci-test.sh`
+  * KUBEBUILDER_ENVTEST_KUBERNETES_VERSION: `1.24.2`
```

Comment:
Let's update the other KUBEBUILDER_ENVTEST_KUBERNETES_VERSION below to v1.24.2

Reply:
Done